text/0000-serialization.md

```diff
@@ -0,0 +1,187 @@
+- Start Date: 2015-10-16
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+The goal of this RFC is to formalize and document the various serialization formats used by Refract, as well as explicitly list how to handle ambiguity in refracted metadata and attributes.
+
+# Motivation
+
+1. Currently they are not defined in the specification. There is one RFC for Embedded Refract but it is not visible to people looking at the Refract specification repository.
+2. Optionally refracted element metadata and attributes cause ambiguity issues with some serialization formats. We need to formally document how to avoid these ambiguities.
```

Comment:
@danielgtaylor Will this be resolved with this?

https://github.com/refractproject/rfcs/pull/29

If so, would you care to update this RFC accordingly?